runtimes/core/src/lib.rs

```diff
@@ -266,6 +282,41 @@ impl Runtime {
 
         log::set_tracer(tracer.clone());
 
+        // Find push subscriptions which should be proxied to the subscribing service by the gateway
+        let proxied_push_subs = resources
+            .pubsub_clusters
+            .iter()
+            .flat_map(|c| c.subscriptions.iter())
+            .filter(|s| s.push_only)
+            .filter_map(|s| {
+                let svc_name = (|| -> Result<String, anyhow::Error> {
+                    Ok(md
+                        .pubsub_topics
+                        .iter()
+                        .find(|t| t.name == s.topic_encore_name)
+                        .context("could not find topic")?
+                        .subscriptions
+                        .iter()
+                        .find(|ms| ms.name == s.subscription_encore_name)
+                        .context("could not find sub")?
```

Comment:
Should we fail runtime startup in all these cases? Or is it better to fail when handling the push?

Reply:
Figured it would be a case of broken runtime config when there's a subscription defined which is not available in the metadata, so opted for failing. But I'm easily convinced so I'll change it :)